src/hotspot/share/classfile/vmIntrinsics.cpp

```diff
@@ -36,31 +36,31 @@
 #include "utilities/tribool.hpp"
 
 // These are flag-matching functions:
-inline bool match_F_R(jshort flags) {
+inline bool match_F_R(u2 flags) {
```

Comment:
I wish more code could be size-agnostic.  So instead of using `u2` here, there could be a typedef in accessFlags.hpp that we could use that hides the size.  However, it's not a big deal, because it seems unlikely this type will change much in the future without a JVM spec change.  I'm tempted to suggesting using AccessFlags here, but it's a class.  Since this is an end-point "consumer" of the type that doesn't store it or pass it along, we could even use something like `uint` here.

Reply:
That seems like a good idea if this weren't so limited.  I did chat with DanH. to see if access flags will ever grow in size from u2 and he was doubtful that would ever happen. uint would work here too, but this small use might prevent some future -Wconversion warnings and doesn't hurt anything.
Thank you for reviewing.